src/app/report/difference-modal/difference-modal.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Component, Output, TemplateRef, ViewChild } from '@angular/core';
+import { ReportDifference } from '../../shared/interfaces/report-difference';
+import { TitleCasePipe } from '@angular/common';
```

Comment:
Unused import?

Reply:
No it's being used, look at line 42 in the html.